static/app/views/feedback/list.tsx

```diff
@@ -1,17 +1,28 @@
-import Alert from 'sentry/components/alert';
+import {browserHistory, RouteComponentProps} from 'react-router';
+import styled from '@emotion/styled';
+
+import DatePageFilter from 'sentry/components/datePageFilter';
+import EnvironmentPageFilter from 'sentry/components/environmentPageFilter';
+import FeedbackTable from 'sentry/components/feedback/table/feedbackTable';
 import useFetchFeedbackList from 'sentry/components/feedback/useFetchFeedbackList';
 import * as Layout from 'sentry/components/layouts/thirds';
+import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
 import PageFiltersContainer from 'sentry/components/organizations/pageFilters/container';
 import {PageHeadingQuestionTooltip} from 'sentry/components/pageHeadingQuestionTooltip';
-import Placeholder from 'sentry/components/placeholder';
+import Pagination from 'sentry/components/pagination';
+import ProjectPageFilter from 'sentry/components/projectPageFilter';
 import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
 import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import type {FeedbackListQueryParams} from 'sentry/utils/feedback/types';
 import useOrganization from 'sentry/utils/useOrganization';
 
-export default function List() {
+interface Props extends RouteComponentProps<{}, {}, FeedbackListQueryParams> {}
+
+export default function List({location}: Props) {
```

Comment:
[l]ow: generally like to see component names w/ more context in the name (e.g. FeedbackList) as it's a bit more helpful when debugging w/ react dev tools, but we don't have a rule for this (and also hard to have a lintable rule) so low logaf.

Reply:
ya, this name sucks. I can pick a unique one to represent it better